packages/connectkit/src/constants/supportedChains.tsx

```diff
@@ -1,8 +1,7 @@
 import { ReactNode } from 'react';
 import Logos from './../assets/chains';
 
-type Chain = { id: number; name: string; logo: ReactNode };
-const supportedChains: Chain[] = [
```

Comment:
Any reason why this type has been removed?

Reply:
as const already infers it, so I thought its unnecessary